egs/tedlium/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# This is based on:
+# swbd/s5c/local/nnet3/run_ivector_common.sh and
+# tedlium/s5/local/online/run_nnet2_ms_perturbed.sh
+# see the chain docs for general direction on what training is doing!
+
+. ./cmd.sh
+set -euo pipefail
```

Comment:
better not use set -e. It is more informative to print an error message before exit.